### PR TITLE
stable/memcached: Add support for metrics

### PR DESCRIPTION
**Pull request by olemarkus**
**What this PR does / why we need it**:

This pull request adds memcached_exporter as a sidecar container to the memcached pod.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by olemarkus** [comment]
Ping @gtaylor

File: stable/memcached/templates/statefulset.yaml [hunk 1]
```diff
@@ -66,6 +66,16 @@ spec:
...
 {{ toYaml .Values.resources | indent 10 }}
+{{- if .Values.metrics.enabled }}
+      - name: metrics
+        image: {{ .Values.metrics.image }}
+        imagePullPolicy: {{ default "" .Values.metrics.imagePullPolicy | quote }}
+        ports:
+        - name: metrics
+          containerPort: 9150
+        resources:
+{{ toYaml .Values.metrics.resources | indent 10 }}
```
Comment (paulczar) [inline stable/memcached/templates/statefulset.yaml]: add `metrics.resources` to the README values table.

File: stable/memcached/values.yaml [hunk 2]
```diff
@@ -35,6 +39,24 @@ resources:
     memory: 64Mi
     cpu: 50m
 
+metrics:
+  ## Expose memcached metrics in Prometheus format
+  enabled: false
+
+  ## Memcached exporter image and tag
+  image: quay.io/prometheus/memcached-exporter:v0.4.1
```
Comment (paulczar) [inline stable/memcached/values.yaml]: please add to README values table.

**Review by paulczar — CHANGES_REQUESTED** [review]
Please bump the chart version and address the inline comments.

**Pull request closed without merging**